### DISCUSSION TITLE
Orientation Setting using screens

### DISCUSSION THREAD

**Discussion by IsaacKwanMM** [firemodels/fds#13850] [Q&A] [answered]
Hello,

I am currently trying to use a screen to control the airflow out of an enclosure. (FDS 6.7.7) i.e. (found in guidelines)
**_&INIT N_PARTICLES_PER_CELL=1, CELL_CENTERED=T, PART_ID='SCREEN',
XB=1.01,1.02,0.0,1.0,0.0,1.0/
&PART ID='SCREEN', DRAG_LAW='SCREEN', FREE_AREA_FRACTION=0.4, STATIC=T,
SURF_ID='SCREEN', ORIENTATION(1:3,1)=1,0,0 /
&SURF ID='SCREEN', THICKNESS=0.00015, GEOMETRY='CYLINDRICAL',
MATL_ID='ALUMINUM' /
&MATL ID='ALUMINUM', DENSITY=2700., CONDUCTIVITY=200., SPECIFIC_HEAT=0.9 /_**

I know that if I want to control the flow out of the roof, I can use "ORIENTATION(1:3,1)=0,0,1"
But I am not sure about in the x direction, may I use "ORIENTATION(1:3,1)=-1,0,0" or "ORIENTATION(1:3,2)=-1,0,0"
Thanks for the help in advance, I found the guideline a little bit confusing.
![image](https://github.com/user-attachments/assets/09d6c570-5e0f-4d23-bd1f-8f1afa379361)

**Comment by mcgratta** [comment]
This feature of splitting a cylinder has been removed from the latest version. There were too many inconsistencies with this approach and we decided not to support it any more. I suggest you use the latest version of FDS.

**Reply by drjfloyd** [reply, marked as answer]
In older versions of FDS, screen drag only uses the first vector entry for ORIENTATION. Screen drag is meant to represent a window screen which is a plane. The ORIENTATION for screen drag is defning the normal vector to the screen. The drag imposed by the screen is apportioned to the three coordinate directions based on the absolute value of the components of the unit normal vector so 1,0,0 is equivalent to -1,0,0.
